### PR TITLE
Force nightlies to be -pre

### PR DESCRIPTION
**Pull request by hawkrives**
Should fix iTC's issue with us uploading versions below the released version to testflight…

### CODE REVIEW

File: fastlane/lib/versions.rb [hunk 1]
```diff
@@ -46,15 +46,15 @@ def propagate_version(**args)
 
   UI.message "Propagating version: #{version}"
   UI.message 'into the Info.plist and build.gradle files'
+  
+  version = "#{version.split('-')[0]}-pre" if should_nightly?
```
Comment (rye) [inline fastlane/lib/versions.rb:50]: What happens if there is not a `-` in the version for some reason?

Comment (rye) [reply fastlane/lib/versions.rb:50]: If this case is already handled by `should_nightly?`, then that's okay.  I suppose this split won't be performed if `should_nightly?` is false.

Comment (rye) [reply fastlane/lib/versions.rb:50]: I guess my question is really: is it possible that `should_nightly?` will be truthy in a situation where we have a tag without a `-`?  I'm thinking if we release a new production build right around midnight.

Comment (hawkrives) [reply fastlane/lib/versions.rb:50]:
```
irb(main):001:0> '2.0.0'.split('-')[0]
=> "2.0.0"
```

We'll still append `-pre`, so it'll be fine.

Comment (rye) [reply fastlane/lib/versions.rb:50]: Oh true, I had a lapse in judgement about how split is handled.  Split, finding no delimiters, just returns the string it was called on 😄.

**Review by rye — APPROVED** [review]
:shipit:

**Pull request merged**